papis/docmatcher.py

```diff
@@ -45,21 +47,20 @@ def return_if_match(
         >>> from papis.database.cache import match_document
         >>> doc = papis.document.from_data(dict(title='einstein'))
         >>> DocMatcher.set_matcher(match_document)
-        >>> DocMatcher.parse('einste')
-        ([(['einste'], {})], {})
+        >>> result = DocMatcher.parse('einste')
```

Comment:
From the little digging I've done one this, it seems like `pyparsing.ParseResults` used to be printed like above, but now it's
```
ParseResults([ParseResults(['einste'], {})], {})
```
and I haven't managed to convince the doctest to ignore the differences :\